boards/arm/reel_board/Kconfig.board

```diff
@@ -1,9 +1,13 @@
 # Kconfig - reel board configuration
 #
-# Copyright (c) 2018 Phytec Messtechnik GmbH
+# Copyright (c) 2018-2019 PHYTEC Messtechnik GmbH
 #
 # SPDX-License-Identifier: Apache-2.0
 
 config  BOARD_REEL_BOARD
```

Comment:
I would have expected to see `choice` here to exclude selecting both, but somehow that seems to work (at least I can't make menuconfig select both).  Kconfig magic, I guess.